File: app/src/main/java/dev/thehale/papermc_plugin_template/commands/SheepMergeCommandModule.java
package dev.thehale.papermc_plugin_template.commands;

import java.util.List;

import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public interface SheepMergeCommandModule {
    String root();

    boolean execute(Player player, String[] args);

    List<String> tabComplete(CommandSender sender, String[] args);
}

File: app/src/main/kotlin/dev/x208/sheepmerge/commands/SheepMergeCommandModule.kt
package dev.x208.sheepmerge.commands

import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

interface SheepMergeCommandModule {
    fun root(): String

    fun execute(player: Player, args: Array<String>): Boolean

    fun tabComplete(sender: CommandSender, args: Array<String>): List<String>
}
